console/my-account/troubleshooting/cant-log-into-my-account.mdx

```diff
@@ -0,0 +1,53 @@
+---
+meta:
+  title: I cannot log into my account
+  description: This page helps you log into your account
+content:
+  h1: I cannot log into my account
+  paragraph: This page helps you log into your account
+dates:
+  validation: 2022-10-19
+  posted: 2022-10-19
+---
+
+Sometimes you may have problems when connecting to your Scaleway account. If you have encountered any of the following possible cases, check out our solutions.
+
+## Unrecognized IP address error 
+You may try to log into your Scaleway account and see a message like the following: 
+
+<Lightbox src="scaleway-unrecognized-ip.webp" alt="" />
+
+This means you tried to log into your account from an unknown IP address. As a security measure, we require you to use a magic link to complete authentification. 
+ 
+1. Check your inbox for the latest email sent by the Scaleway team.
+2. Click on the link provided in the email. It should be a button titled **"Authenticate Myself"**.
+3. You're safely logged into your Scaleway account. 
+
+
+## I have forgotten my password
+
+If you forgot or lost your password, you need to reset it. There are several methods to retrieve it :
+  - [Passwordless authentification using a Magic Link](https://www.scaleway.com/en/docs/console/my-account/how-to/recover-a-lost-password/#passwordless-authentication-using-a-magic-link)
+  - [Password reset using email validation](https://www.scaleway.com/en/docs/console/my-account/how-to/recover-a-lost-password/#password-reset-using-email-validation)
+  - [Password recovery in case of lost email account](https://www.scaleway.com/en/docs/console/my-account/how-to/recover-a-lost-password/#password-recovery-in-case-of-lost-email-account)
```

Comment:
```suggestion
  - [Password recovery in case of lost email account](/console/my-account/how-to/recover-a-lost-password/#password-recovery-in-case-of-lost-email-account)
```